cirq/aqt/aqt_sampler.py

```diff
@@ -25,20 +25,35 @@
 import json
 import time
 import uuid
-from typing import Iterable, List, Union, Tuple, Dict, cast, TYPE_CHECKING
+from typing import List, Union, Tuple, Dict, cast, TYPE_CHECKING
 
 import numpy as np
 from requests import put
-from cirq import circuits, Sampler, resolve_parameters, LineQubit
-from cirq.study.sweeps import Sweep
-from cirq.aqt.aqt_device import AQTSimulator, get_op_string
-from cirq import study, ops, IonDevice
+
+from cirq import Circuit
+
+from cirq.work import Sampler
+
+from cirq.study import ParamResolverOrSimilarType, Sweepable, \
+    TrialResult, to_resolvers
+
+from cirq.ops import GateOperation, EigenGate
+
+from cirq.protocols import resolve_parameters
+
+from cirq.ion import IonDevice
+
+from cirq.devices import LineQubit
+
+from cirq.aqt.aqt_device import AQTSimulator, get_aqt_device_op_string
```

Comment:
Unneeded whitespace.  Also, I think these imports were fine before.

The problem is importing all of cirq with `import cirq`  
Importing ops should be fine.  Casey's test should prevent more accidental circular imports

Reply:
If it is fine, I will keep it like this. I thought that importing ops is also too big. I would prefer not to change everywhere in the code the occurences.